### PR TITLE
Add emails for fake.is-a.dev

### PR DESCRIPTION
**Pull request by fakename131**

### CODE REVIEW

**Review by hackermondev — APPROVED** [review]

**Review by hackermondev — DISMISSED** [review]
IIRC you cannot use ``CNAME`` record with MX record, only an A record with MX record is allowed.

**Comment by YEEETT** [comment]
Hopefully that works. (I modified my fork and I think I saw it updated here, but I may be wrong because I don't have much experience with pull requests)

**Review by a69y — CHANGES_REQUESTED** [review]
Remove CNAME & Add Array Of 'A' Records For Github Pages
image

Look At This PR For Reference :- https://github.com/is-a-dev/register/pull/1409

File: domains/fake.json [hunk 1]
```diff
@@ -6,6 +6,11 @@
     "email": "fake@fake.is-a.dev"
   },
   "record": {
-    "CNAME": "yeeett.github.io"
+    "A": "185.199.108.153",
```
Comment (MaskDuck) [inline domains/fake.json:9]:
```suggestion
    "A": ["185.199.108.153"],
```

**Review by MaskDuck — CHANGES_REQUESTED** [review]

**Comment by YEEETT** [comment]
> Remove CNAME & Add Array Of 'A' Records For Github Pages image
> 
> Look At This PR For Reference :- #1409

The first one was the IP address I used.

**Review by MaskDuck — APPROVED** [review]
not sus

**Review by TheBotlyNoob — APPROVED** [review]
yep

**Pull request merged**